### PR TITLE
Backporting for 2.426.1

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Backporting for LTS 2.426.1

Latest core version: [Jenkins 2.429](https://www.jenkins.io/changelog/#v2.429)

### Fixed

<pre>
JENKINS-72248           Minor                   2.428
        Backport uptime telemetry
        https://issues.jenkins.io/browse/JENKINS-72248

JENKINS-72189           Major                   2.430
        Drag&Drop for repeatables not working for existing entries (regression in 2.335)
        regression
        https://issues.jenkins.io/browse/JENKINS-72189

JENKINS-72179           Minor                   2.429
        Parameters view of builds does not show descriptions of checkbox parameters (regression in 2.179)
        regression
        https://issues.jenkins.io/browse/JENKINS-72179

JENKINS-72170           Major                   2.428
        Cannot add additional branch sources in multibranch pipeline after 2.422
        regression
        https://issues.jenkins.io/browse/JENKINS-72170

JENKINS-72163           Major                   3160.vd76b_9ddd10cc
        Retry on initial connection failure occurs in one entrypoint but not the other
        https://issues.jenkins.io/browse/JENKINS-72163

JENKINS-72156           Minor                   2.428, 2.414.3
        Include Jetty 10.0.17 in a Jenkins release
        https://issues.jenkins.io/browse/JENKINS-72156

JENKINS-72020           Minor                   2.428
        Unable to reorder clouds anymore via UI
        https://issues.jenkins.io/browse/JENKINS-72020

JENKINS-71937           Major                   2.427
        Jenkins 2.421 breaks vSphere Cloud agent labels
        regression
        https://issues.jenkins.io/browse/JENKINS-71937

JENKINS-71252           Critical                2.427
        Form validation is never displayed for fields initially hidden under Advanced section (regression in 2.355)
        regression
        https://issues.jenkins.io/browse/JENKINS-71252

JENKINS-70793           Minor                   2.427
        Form validation does not clear previous errors (regression in 2.355)
        regression
        https://issues.jenkins.io/browse/JENKINS-70793

JENKINS-65368           Major                   3159.vb_8c0ef2b_55a_f as included in 2.428
        Remoting agent.jar does not work behind proxy
        https://issues.jenkins.io/browse/JENKINS-65368
</pre>

The developer mailing list has the [backporting announcement](https://groups.google.com/g/jenkinsci-dev/c/d9Qr08oFstQ/m/-d8IzXwUNgAJ).  The [2.426.1 release checklist](https://github.com/jenkins-infra/release/issues/464) has more information as well.

Several candidates are not included in this pull request, but the authors of those candidates have been asked if those changes should be included in the 2.426.1 LTS backports.  Those include:

* Fix drag and drop for repeatables - [JENKINS-72189](https://issues.jenkins.io/browse/JENKINS-72189) & [JENKINS-71089](https://issues.jenkins.io/browse/JENKINS-71089)

### Testing done

Ran the generated war file interactively with Java 11, Java 17, and Java 21 and confirmed that it was well behaved with a few minutes of interactive testing.

Ran tests successfully with `mvn clean verify` on Linux.

### Proposed changelog entries

N/A - changelog will be created by the documentation team.

### Proposed upgrade guidelines

N/A - no items in this pull request that need an entry in the upgrade guide

### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@NotMyFault

Before the changes are marked as `ready-for-merge`:

### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

### CODE REVIEW

**Review by basil — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Comment by MarkEWaite** [comment]
Based on the [answer](https://github.com/jenkinsci/jenkins/pull/8492#issuecomment-1784203056) from @timja I've added e6b453b0b42c81963ccfff61dafd9ef00786006d , a backport of the fix for [JENKINS-72020 - Restore ability to reorder clouds](https://issues.jenkins.io/browse/JENKINS-72020)

**Comment by MarkEWaite** [comment]
@daniel-beck marked two issues as LTS candidates so they are included as 508064372295e4382e81d64f28338e524590c4c6 and b5607975c048dd74306d49fecdc57eefa8341b7f

File: pom.xml [hunk 1]
```diff
@@ -86,7 +86,7 @@ THE SOFTWARE.
     <changelog.url>https://www.jenkins.io/changelog</changelog.url>
 
     <!-- Bundled Remoting version -->
-    <remoting.version>3148.v532a_7e715ee3</remoting.version>
+    <remoting.version>3160.vd76b_9ddd10cc</remoting.version>
```
Comment (jglick) [inline pom.xml:89]: FTR https://github.com/jenkinsci/remoting/compare/3148.v532a_7e715ee3...3160.vd76b_9ddd10cc = https://github.com/jenkinsci/remoting/pull/675 plus some unrelated changes; see discussion in https://issues.jenkins.io/browse/JENKINS-72163

Comment (MarkEWaite) [reply pom.xml:89]: Thanks for pointing to that difference!  I had missed that there is an additional bug fix that is included in remoting 3160 as delivered in Jenkins 2.428.  The additional fix is [JENKINS-65368 Remoting agent.jar does not work behind proxy](https://issues.jenkins.io/browse/JENKINS-65368).  I've labeled that bug report as `2.426.1-fixed` in Jira and included it in the pull request description
